.github/CODEOWNERS

```diff
@@ -27,6 +27,7 @@ tests/debugging/                    @DataDog/debugger-python
 ddtrace/appsec/                     @DataDog/asm-python
 tests/appsec/                       @DataDog/asm-python
 tests/contrib/flask/test_flask_appsec.py    @DataDog/asm-python
+tests/contrib/flask/test_flask_appsec_telemetry.py    @DataDog/asm-python
```

Comment:
I think this should be covered by this line below:
```
tests/contrib/*/test*appsec*.py @DataDog/asm-python
```

Reply:
:+1: